src/renderers/shared/stack/event/ReactSyntheticEventType.js

```diff
@@ -8,14 +8,29 @@
  *
  * Flow type for SyntheticEvent class that includes private properties
  *
- * @providesModule ReactSyntheticEvent
+ * @providesModule ReactSyntheticEventType
  * @flow
  */
 
 'use strict';
 
-export type DispatchConfig = any;
+import type {ReactInstance} from 'ReactInstanceType';
+
+export type DispatchConfig = {
+  dependencies: Array<string>,
+  phasedRegistrationNames?: {
```

Comment:
👍 

(FYI, I have no idea what this code is doing so I'm really asking questions here, you are probably writing the correct type definition :))